modules/oadp-1-4-1-release-notes.adoc

```diff
@@ -0,0 +1,126 @@
+link:https://issues.redhat.com/browse/OADP-4172[OADP-4172]
+
+.OADP Velero can now retry updating the restore status if initial update fails
+
+Previously, OADP Velero failed to update the restored CR status and the status was stuck at `InProgress` indefinitely. As a consequence, it affected other components' behavior that relied on the backup and restore CR status to determine the completion. 
+With this update, the restore CR status would not be `InProgress` indefinitely and should eventually be `Completed` or `Failed`.
+link:https://issues.redhat.com/browse/OADP-3227[OADP-3227]
+
+.Restoring BuildConfig Build from a different cluster is successful without any errors
+ 
+Previously, on performing a restore of the `BuildConfig` Build resource from a different cluster, the application resulted in errors on TLS verification to the internal image registry. It resulted in the `failed to verify certificate: x509: certificate signed by unknown authority` error.
+With this update, the application restores successfully without any errors.
+link:https://issues.redhat.com/browse/OADP-4692[OADP-4692]
+
+.Restoring an empty PVC is successful 
+
+Previously, downloading data failed while restoring an empty persistent volume claim (PVC). It failed with the following error:
+----
+data path restore failed: Failed to run kopia restore: Unable to load
+    snapshot : snapshot not found
+----
+With this update, an empty PVC is restored correctly and data download does not fail.
+link:https://issues.redhat.com/browse/OADP-3106[OADP-3106]
+
+.There is no Velero memory leak in CSI and DataMover plugins
+
+Previously, Velero memory leak was caused while using the CSI and DataMover plugins. When the backup ended, the Velero plugin instance was not deleted and it caused the memory to grow until an Out of Memory (OOM) condition for the Velero pod. With this update, there is no Velero memory leak.
+link:https://issues.redhat.com/browse/OADP-4448[OADP-4448]
+
+.Post restore hooks does not start before the related PVs are released
+
+Previously, due to the asynchronous nature of the Data Mover operation, a post-hook might be attempted before the related pods' persistent volumes (PVs) were released by the Data Mover persistent volume claim (PVC). As a consequence, backup PartiallyFailed.
+With this update, the post-hook operation is not started now before the related PVs are released by data Mover PVC, and the backup is successful. 
+link:https://issues.redhat.com/browse/OADP-3140[OADP-3140]
+
+.Deploying a DPA works as expected in namespaces with more than 37 characters
+
+While installing the OADP Operator in a namespace with more than 37 characters and when creating a new DPA, labeling the "cloud-credentials" Secret fails, and the DPA reports the following error:
```

Comment:
```suggestion
While installing the OADP Operator in a namespace with more than 37 characters to create a new DPA, labeling the "cloud-credentials" Secret will fail and the DPA reports the following error:
```

Reply:
Suggesting simplifications because this is a complicated sentence. Emphasizing that the "37 characters" creates the problem and not creating a new DPA.

Reply:
https://www.ibm.com/docs/en/ibm-style?topic=medium-global-audiences